### ISSUE TITLE
Allow programmatic command invocation

### ISSUE THREAD

**Issue by T99** [elijahjcobb/keybase-bot-builder#2]
The ability to programmatically execute commands that are currently only accessible by invoking them by messaging the bot would be a nice addition to allow for easier command scheduling.

Ideally, something to the effect of:
```typescript
bot.invoke("command and parameters")
```
So that commands could be invoked just as they would via the messaging system. This would also obviously mean much less work as you could probably just use your in-place command parsing system.

**Comment by elijahjcobb** [comment]
Like it, will do! :smile:

**Comment by elijahjcobb** [comment]
So I have implemented this. In order for the bot to know the context you are going to have to pass the message object but you can save a reference to the one you get when the command needs to be set up. It just replaces the message you _"invoke"_ in the context's text field also, the sent time is updated to simulate it is current.

The new method's signature is:
```typescript
invoke(command: string, message: KBMessage): Promise<void>
```

Testing right now and will make updated version later today.

**Comment by elijahjcobb** [comment]
Available in `v1.1.9` on NPM.
